src/vc.ts

```diff
@@ -269,6 +273,100 @@ export async function updateAddProof(
     return vc;
 }
 
+export async function updateEcdsaSecp256k1Proof(
+    oldStmt: StatementUri,
+    vc: any,
+    callbackFn: SignCallback,
+    issuerDid: Cord.DidDocument,
+    network: ApiPromise,
+    options: any,
+) {
+    if (options.type === 'affinidi') {
+        delete vc.credentialHash;
+        // Add statement as id in VC
+        const vcId = oldStmt.split(':').slice(0, 3).join(':');
```

Comment:
may be this will need to keep the changes as is? they need a ':' in the ID? If not, then OK. Because at least Inji code needed some ':' in ID.

Reply:
I tried it with keeping the digest in the statement, but it was not getting verfied in affinidi portal.
Hence, removed the digest from statement